src/App.js

```diff
@@ -1,28 +1,8 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
```

Comment:
- [ ] Greate job refactoring the Calculator component however, you need to use class-based components throw  your app, kindly also make the App a class component